__tests__/app.test.js

```diff
@@ -45,4 +46,37 @@ describe("app", () => {
         });
     });
   });
+
+  describe("GET - /api/articles/:article_id", () => {
+    test("status: 200 - should return return an article object with all the containing all expected properties ", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((res) => {
+          expect(res.body.article).toEqual({
```

Comment:
I would still use `expect.objectContaining` here to account for additional properties in the future

Reply:
Hi Dave, I've recommitted with this added. This ok?